src/hooks/usePokemons.ts

```diff
@@ -32,8 +45,30 @@ const usePokemons = () => {
       };
     }) ?? [];
 
+  const handleLoadMore = async () => {
```

Comment:
in general, if you return function from hook -> it is always good idea to use `useCallback`